presto-main/src/main/java/com/facebook/presto/operator/GroupByHash.java

```diff
@@ -46,10 +59,22 @@ static GroupByHash createGroupByHash(
             boolean processDictionary,
             JoinCompiler joinCompiler)
     {
+        return createGroupByHash(hashTypes, hashChannels, inputHashChannel, expectedSize, processDictionary, joinCompiler, x -> true);
+    }
```

Comment:
We generally avoid overloads. Sometimes, we have overloads. However, we try to make sure the number of overloads are linear (not exponential). For example,
* f(a, b), f(a, b, c) is ok
* f(a, b), f(a, c), f(a, b, c) is not ok.

There is only one caller to this overload. Inline this overload (line 54, not line 62)